Two versions of the same crate when using lunarity as a library

With the new organization there is an issue with using lunarity as a library.

I wrote some code to illustrate the issue: 

```rust
extern crate lunarity;

use lunarity::ast::{Program};
use std::{io::Read, fs};

fn main() {

    let mut src = String::new();
    fs::File::open("simple.sol").unwrap().read_to_string(&mut src).unwrap();
    let ast: Program = lunarity::parse(&src).unwrap();
}
```

`cargo tree` output

```
lunarity-tests v0.1.0 (file:///home/insi/Projects/EDB/lunarity-tests)
└── lunarity v0.1.0 (file:///home/insi/Projects/EDB/lunarity/lunarity)
    ├── lunarity-ast v0.1.0 (file:///home/insi/Projects/EDB/lunarity/ast)
    │   └── toolshed v0.4.0
    │       └── fxhash v0.2.1
    │           └── byteorder v1.2.6
    ├── lunarity-lexer v0.1.0 (file:///home/insi/Projects/EDB/lunarity/lexer)
    │   └── toolshed v0.4.0 (*)
    └── lunarity-parser v0.1.0 (file:///home/insi/Projects/EDB/lunarity/parser)
        ├── lunarity-ast v0.1.0
        │   └── toolshed v0.4.0 (*)
        ├── lunarity-lexer v0.1.0
        │   └── toolshed v0.4.0 (*)
        └── toolshed v0.4.0 (*)
```

Error: 
```
   Compiling lunarity-tests v0.1.0 (/home/insi/Projects/EDB/lunarity-tests)
error[E0308]: mismatched types
  --> src/main.rs:10:24
   |
10 |     let ast: Program = lunarity::parse(&src).unwrap();
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `lunarity::ast::Program`, found struct `lunarity_ast::Program`
   |
   = note: expected type `lunarity::ast::Program<'_>`
              found type `lunarity_ast::Program<'_>`
note: Perhaps two different versions of crate `lunarity_ast` are being used?
  --> src/main.rs:10:24
   |
10 |     let ast: Program = lunarity::parse(&src).unwrap();
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: Could not compile `lunarity-tests`.

To learn more, run the command again with --verbose.
```

This is because the `lunarity` package is using the local file paths for the `ast`, `lexer` and `parser` dependencies, while `ast`, `lexer`, and `parser` dependencies are using the crates from crates.io

--- a/lunarity/Cargo.toml
+++ b/lunarity/Cargo.toml
@@ -6,9 +6,9 @@ license = "GPL-3.0"
 repository = "https://github.com/paritytech/lunarity"
 
 [dependencies]
-lunarity-ast = { path = "../ast" }
-lunarity-lexer = { path = "../lexer" }
-lunarity-parser = { path = "../parser" }
+lunarity-ast = "0.1.0"
+lunarity-lexer = "0.1.0"
+lunarity-parser = "0.1.0"
 
 [dev-dependencies]
 toolshed = "0.4"

